src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,17 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+  isLoading: true,
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    checkStatus: () => 'Under construction',
+  },
```

Comment:
- [ ] Kindly modify your reducer to have a state value and a payload. Currently, you're just returning a value that is not passed to the state and the value returned needs to be stored in the state variable. You could implement it this way:
```javascript
checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    }```

Reply:
@AmaduKamara I tried this code but it's giving linters error.

![Screenshot (21)](https://user-images.githubusercontent.com/117263981/229747315-fbfecb40-56b7-4945-86e4-c74ae0caae96.png)


kindly suggest, what should I do to make it correct.